watsononlinestore/watson_online_store.py

```diff
@@ -105,15 +112,30 @@ def __init__(self, bot_id, slack_client,
         self.delay = 0.5  # second
 
     def context_merge(self, dict1, dict2):
+        """Combine 2 dicts into one for Watson Conversation context.
+
+        Common data in dict2 will override data in dict1
+
+        :param: dict1 original context dictionary
```

Comment:
<colon>param [type] <param-name><colon> <desc...>
I think the type is optional, but I guess we want them.